components/partners.js

```diff
@@ -14,7 +14,7 @@ const Partners = () => {
               <a href={partner.website} target="_blank" rel="noreferrer">
               <img
                 src={partner.logo}
-                className={`img-fluid partners-logo ${partner.paddingClass}`}
```

Comment:
Please revert this. The logos are currently stretched.

Reply:
Oops forgot to add the style.

Reply:
added